### PR TITLE
CC-869 Add ecs-image-build/

### PR DESCRIPTION
**Pull request by NeilShadrach**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: ecs-image-build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-17:latest
```
[the remark this answers is not in the document]
Comment (NeilShadrach) [reply ecs-image-build/Dockerfile]: Good spot! I'm glad one of us is awake

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by mike-kelly-ch — APPROVED** [review]
👍

**Pull request merged**